htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -205,6 +205,8 @@ INSERT INTO llx_payment_vat (rowid, fk_tva, datec, datep, amount, fk_typepaiemen
 --ALTER TABLE llx_tva DROP COLUMN fk_bank;
 
 ALTER TABLE llx_tva ALTER COLUMN paye SET DEFAULT 0;
+-- VMYSQL4.3 UPDATE llx_tva as tva INNER JOIN llx_bank b ON (b.rowid = tva.fk_bank) SET tva.fk_account = b.fk_account WHERE tva.fk_account IS NULL;
```

Comment:
Column tva.fk_bank does not exists anymore in v14

Reply:
@eldy OK, I thought it could be interesting but no problem, it's not very important